src/style.css

```diff
@@ -0,0 +1,106 @@
+@font-face {
+  font-family: "Avenir";
+  src: url("./fonts/Avenir-Book.ttf") format(ttf);
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url("./fonts/Avenir-Heavy.ttf") format(ttf);
+  font-weight: 700;
+}
+
+html {
+  font-family: "Avenir", sans-serif;
+  box-sizing: border-box;
+  margin: 0;
+}
+
+/* Big Search Bar */
+
+.search__bar {
+  width: 100%;
+  display: flex;
+  align-items: center;
+  margin: 20px 0;
+
+  border: 1px solid #e7e4ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+
+  transition: box-shadow 0.3s ease;
+}
+
+.search__input {
+  background-color: white;
+  color: #3d4e61;
+
+  font-size: 16px;
+  font-family: "Avenir", sans-serif;
```

Comment:
use `inherit` you have already added font-family for HTML
https://developer.mozilla.org/en-US/docs/Web/CSS/inherit